### DISCUSSION TITLE
v4 에서 프로필 변경하는법을 모르겠습니다.

### DISCUSSION THREAD

**Discussion by lee-jox** [storycraft/node-kakao#756] [Q&A]
openUploadApiClient.buildProfile 내에 api 를 사용할경우 

axios 통신에서 404 에러가 나오네요. 프로필사진도 같이 변경하고자 합니다.

api 를 분석해서 개별로 테스트해도 제 실력으로는 원인을 찾기힘든데요

프로필 변경 가능한걸까요~ 아니면 막힌걸까요 ^^;

**Comment by storycraft** [comment]
노드카카오에 있는 프로필 사진을 업로드하는 api가 오래된 api 였거나 다른 엔드포인트로 바뀌었을 가능성이 있습니다. 프로필 이름은 buildProfile메서드 없이도 변경 가능합니다.
